ci/long_running_tests/workloads/serve_scalability.py

```diff
@@ -0,0 +1,67 @@
+# A multi-node scalability test. We put an http proxy on the head node and spin
+# up e.g. 20 nodes and put a bunch of replicas on the cluster, and run a stress
+# test.
+
+import logging
+import time
+import subprocess
+import requests
+
+import ray
+from ray import serve
+from ray.serve import BackendConfig
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+ray.init(address="auto")
+client = serve.start()
+
+# These numbers need to correspond with the autoscaler config file.
+# The number of remote nodes in the autoscaler should upper bound
+# these because sometimes nodes fail to update.
```

Comment:
:(

Reply:
Haha, actually this is a holdover from `test_many_tasks.py` from 2 years ago.  I haven't experienced the problem they're referencing yet so maybe it's fixed, but I thought I'd leave the comment in just to be safe.